domain/lib/src/usecases/myspace/upload_file_interactor.dart

```diff
@@ -55,8 +55,7 @@ class UploadFileInteractor {
           return Left(UploadFileFailure(Exception()));
         }, (success) {
           if (success is FileUploadProgress) {
```

Comment:
TBH, I am not happy with it `if-else`, I like the way we need to fight with the old way. Fighting!!!